common/v2/features/PurchaseMembership/components/MembershipEducation.tsx

```diff
@@ -1,7 +1,207 @@
 import React from 'react';
+import styled from 'styled-components';
+import { withRouter, Link } from 'react-router-dom';
 
-const MembershipEducation = () => {
-  return <>EDU Panel for MyCrypto Membership!</>;
-};
+import { FullSizeContentPanel, Button, Typography } from 'v2/components';
+import translate from 'v2/translations';
+import { ROUTE_PATHS } from 'v2/config';
+import { BREAK_POINTS, SPACING, COLORS } from 'v2/theme';
+
+import {
+  FullSizePanelSection,
+  SpacedPanelSection,
+  RowPanelSection
+} from '../../../components/FullSizeContentPanel';
```

Comment:
Weird path, you can import from `v2/components`.

Reply:
Weirdly I can't import it from 'v2/components' it always shows me an error